presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -252,6 +252,14 @@ public static Slice md5(@SqlType(StandardTypes.VARBINARY) Slice slice)
         return computeHash(Hashing.md5(), slice);
     }
 
+    @Description("compute murmur3 hash")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice murmur3(@SqlType(StandardTypes.VARBINARY) Slice slice)
```

Comment:
You are suggesting the java function name to be `murmur3_x64_128` and the exposed UDF name to be `murmurhash3_x64_128`. Is the omission of "hash" word in former case intentional ?

Reply:
Actually I messed up it should `murmurhash3X64128` since the rest of the functions are camelcase. Yes it could have hash in the name. The name of the actual function is much less critical because we are assigning the name explicitly.

Reply:
Done